examples/ex1_plot_map.py

```diff
@@ -0,0 +1,45 @@
+"""
```

Comment:
Rename this file to be more informative. If you also want to skip the example, rename it to start `skip_<rest of the name>`